src/styles/mixin.js

```diff
@@ -0,0 +1,18 @@
+import { css } from 'styled-components';
+
+export const flexBox = (jc = 'center', ai = 'center', fd = 'row') => css`
+  display: flex;
+  justify-content: ${jc};
+  align-items: ${ai};
+  flex-direction: ${fd};
+`;
+
+export const positionCenter = (position = 'absolute') => {
+  if (position === ('absolute' & 'fixed')) return;
```

Comment:
이 조건문은 실제 동작이 안될 것 같아보입니다.

```suggestion
  if (!(position === 'absolute' || position === 'fixed')) return;
```